src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantDefendantAgreedMediationRespondentNotificationHandler.java

```diff
@@ -88,7 +88,9 @@ private String addEmail(CaseData caseData) {
 
     private String addTemplate(CaseData caseData) {
         if (caseData.isRespondent1NotRepresented()) {
-            return notificationsProperties.getNotifyRespondentLiPMediationAgreementTemplate();
+            return caseData.isRespondentResponseBilingual() ?
```

Comment:
I's cleaner if you extract it to a function.
Check ClaimantResponseAgreedRepaymentRespondentNotificationHandler.java